YIF.Core.Domain/ApiModels/RequestApiModels/SpecialtyToInstitutionOfEducationPostApiModel.cs

```diff
@@ -13,5 +16,10 @@ public class SpecialtyToInstitutionOfEducationPostApiModel
         /// </summary>
         /// <example>a47e79d8-7e68-43c5-a5c9-1c6dff1e1693</example>
         public string InstitutionOfEducationId { get; set; }
+
+        /// <summary>
+        /// List of payment and education forms
+        /// </summary>  
```

Comment:
Is it ok, that you don't have an example?

Reply:
I think that it's ok.

Reply:
it's ok